docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password-source.md

```diff
@@ -1,32 +1,41 @@
 ---
-id: 1password-source
-title: 1Password Source
-sidebar_label: 1Password
+id: 1password-source  #example: druva-source
+slug: /c2c/1password
+title: 1Password Source #example: Druva Source
+image: 'https://app_icons.s3.amazonaws.com/1Password.png' #replace with your app logo
+sidebar_label: 1Password #example: Druva
+tags:
+  - cloud-to-cloud
+  - 1password  #example: Druva
```

Comment:
```suggestion
id: 1password-source
slug: /c2c/1password
title: 1Password Source
image: 'https://app_icons.s3.amazonaws.com/1Password.png'
tags:
  - cloud-to-cloud
  - 1password
```

Reply:
In this section, remove instances of `#example: Druva`.

Why did you add the slug tag to change the URL of the article from:
> /docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/1password/
to
>/docs/c2c/1password/

Reply:
To simplify the link when creating a new C2C. Instead of using a cid redirection, using a more structured way. Happy to change it back if that makes things easier. What do you think @jpipkin1

Reply:
@yleiferman, as a user I find it disorienting to click a link in the left nav and have it resolve to a different URL than I expect. I'd prefer you change it back.

@kimsauce, what do you think?